plugins/dlab_templates/dlab_templates/infrastructure/controllers.py

```diff
@@ -0,0 +1,11 @@
+from dlab_core.infrastructure.controllers import BaseAPIController
+
+
+class APITemplatesController(BaseAPIController):
+    allowed_types = ['exploratory', 'computational']
+
+    @classmethod
+    def get_templates(cls, type):
+        if type not in cls.allowed_types:
+            return {"code": 0, "message": "string"}, 400
+        return {}, 200
```

Comment:
codes needs to be a constants or you can get them from flask lib

Reply:
added statuses